gui/src/main/daemon-rpc.ts

```diff
@@ -437,6 +437,25 @@ export class DaemonRpc {
     await this.callNumber(this.client.setWireguardMtu, mtu);
   }
 
+  public async setWireguardQuantumResistant(quantumResistant?: boolean): Promise<void> {
+    const quantumResistantState = new grpcTypes.QuantumResistantState();
+    switch (quantumResistant) {
```

Comment:
What about this shorter approach?

```suggestion
    const state = quantumResistant === true ? 'ON'
      : quantumResistant === false ? 'OFF'
        : 'AUTO';

    quantumResistantState.setState(grpcTypes.QuantumResistantState.State[state]);
```

Reply:
I agree that the switch statement is a bit long and clunky but I'm not a huge fan of nested ternary expressions either. We do a lot of convertions to/from enums in this file and I opted for the switch statement to be more consistent with other conversions of bigger enums. I'll keep the switch statement for now.

I will hopefully spend some time refactoring and improving this file as a whole in a not too distant future and improving these convertions will be on my list of things to improve.